source/Cosmos.HAL2/Drivers/Video/VBEDriver.cs

```diff
@@ -76,19 +76,35 @@ private enum EnableValues
         /// <param name="bpp">BPP (color depth).</param>
         public VBEDriver(ushort xres, ushort yres, ushort bpp)
         {
-            if (VBE.IsAvailable())
+            PCIDevice videocard;
+
+            if (VBE.IsAvailable()) //VBE VESA Enabled Mulitboot Parsing
             {
                 Global.mDebugger.SendInternal($"Creating VBE VESA driver with Mode {xres}*{yres}@{bpp}");
                 IO.LinearFrameBuffer = new MemoryBlock(VBE.getLfbOffset(), (uint)xres * yres * (uint)(bpp / 8));
                 lastbuffer = new ManagedMemoryBlock((uint)xres * yres * (uint)(bpp / 8));
             }
-            else
+            else if (Available()) //Bochs Graphics Adaptor ISA Mode
```

Comment:
Can you rename the method so the comment information is contained?

Reply:
Okay